littlepay/api/products.py

```diff
@@ -14,6 +14,16 @@ class ProductResponse:
     description: str
     participant_id: str
 
+    def csv(self) -> str:
+        """Get a CSV str representation of values for this ProductResponse."""
+        return ",".join(vars(self).values())
```

Comment:
Same question here about potential `,`s in any of the values for the product

Reply:
Fixed in [b8187d2](https://github.com/cal-itp/littlepay/pull/19/commits/b8187d229627178b882b898f39eaf635eca15488)